aptos-move/block-executor/src/scheduler.rs

```diff
@@ -610,6 +657,27 @@ impl Scheduler {
         SchedulerTask::NoTask
     }
 
+    pub fn finish_abort_during_commit(&self, txn_idx: TxnIndex) {
+        {
+            // acquire exclusive lock on the validation status of txn_idx, and hold the lock
```

Comment:
the comments are wrong, and we shouldn't need the lock either, decreasing validation index (from here, or from the caller) would be sufficient.

However, I think we shouldn't even decrease validation index from here - so we can get rid of the whole function, and then we can just always decrease the validation index after finishing the execution.